docs/source/command_line_tools/command_line_usage/update_command_line_usage.sh

```diff
@@ -0,0 +1,16 @@
+# This script echoes the command line usage for script tools to files in this directory
```

Comment:
I don't think you should include this (see below) but in future, I'd add a shebang to scripts and make them executable.